src/sage/dynamics/arithmetic_dynamics/wehlerK3.py

```diff
@@ -905,11 +913,11 @@ def degenerate_primes(self,check=True):
             [2, 3, 5, 11, 23, 47, 48747691, 111301831]
         """
         PP = self.ambient_space()
-        if PP.base_ring() in _NumberFields or is_NumberFieldOrder(PP.base_ring()):
-            if PP.base_ring() != ZZ and PP.base_ring() != QQ:
+        if PP.base_ring() != ZZ and PP.base_ring() != QQ:
```

Comment:
I'm not sure of that this change is correct. You changed the order of the tests but not of the error messages...

Reply:
Thanks, I've rephrased it in 869d776fa9b